docs/readme.md

```diff
@@ -1,15 +1,70 @@
-# docs (by Docusaurus)
+# Introduction
```

Comment:
This is a copy of `/README.md` Docusaurus doesn't like doing above it's root directory.  I tried a symlink and the link appears to fail docusaurus's build steps so I just copied to move past the problem as a blocker

Reply:
Does this mean we have a duplicate? If we can't figure this out, we could change the workflow to copy the top level readme in this directory as part of the workflow.

@Amruta-Ranade do you have thoughts on the repo readme vs the docs.airbyte.io readme? We've kept it the same for continuity. I want to continue doing so and was wondering if you have other plans.

Reply:
I do have plans to edit the landing page text. The repo readme is perfect for the OSS audience but I'd like the docs readme to be more user-centric. Keeping them separate makes sense to me.